src/card/src/styles/index.cssr.ts

```diff
@@ -40,6 +40,9 @@ export default c([
       box-shadow .3s var(--bezier),
       border-color .3s var(--bezier);
   `, [
+    c('&:target', {
+      borderColor: 'var(--color-target) !important'
```

Comment:
Must we need `!important`? We can move it to where border-color is need to be overrided.